rwgame/GameWindow.cpp

```diff
@@ -10,7 +10,7 @@ GameWindow::GameWindow() :
 
 void GameWindow::create(const std::string& title, size_t w, size_t h, bool fullscreen)
 {
-	uint32_t style = SDL_WINDOW_OPENGL | SDL_WINDOW_RESIZABLE;
+	uint32_t style = SDL_WINDOW_OPENGL | SDL_WINDOW_RESIZABLE | SDL_WINDOW_HIDDEN;
```

Comment:
nit: While you are at it, SDL2 calls this type `Uint32` which may or may not be the same as `uint32_t`.
(I prefer using intended data types exactly)